chains/polygon/addresses.yaml

```diff
@@ -0,0 +1,21 @@
+aggregationHook: "0x34dAb05650Cf590088bA18aF9d597f3e081bCc47"
```

Comment:
should we omit the quotes?

Reply:
I believe yaml may consider it a number. I didn't put them in explicitly, this is just what the yaml lib spit out